integration_tests/fixtures.py

```diff
@@ -109,7 +75,7 @@ def start_presto(image_tag=None, build=True, with_cache=True):
     return (container_id, run, "localhost", local_port)
 
 
-def wait_for_presto_workers(host, port, timeout=30):
+def wait_for_presto_workers(host, port, timeout=180):
```

Comment:
is this a related change? 
separate commit?

Reply:
It's unrelated and unnecessary. Reverted.